### DISCUSSION TITLE
404 Not Found on login page (clean install)

### DISCUSSION THREAD

**Discussion by TechBreakHelp** [kimai/kimai#2897] [On-Premise: Installation & Updates] [answered]
Hi. I have a problem with the final step of installation.

If I visit https://timetracking.mydomain.com, it redirects to https://timetracking.mydomain.com/login but it shows "404 Not Found". 

var/log/prod.log shows only:
`[2021-11-01 21:53:11] php.WARNING: Warning: preg_match(): Compilation failed: quantifier does not follow a repeatable item at offset 0 {"exception":"[object] (ErrorException(code: 0): Warning: preg_match()>
`
I think there's something wrong with .htaccess. I've already tried to delete kimai2 folder and git clone again, but nothing has changed

**Comment by kevinpapst** [comment, marked as answer]
You can check this here: https://www.kimai.org/documentation/webserver-configuration.html

And you can check your log file at var/log/prod.log

**Reply by TechBreakHelp** [reply]
I've tried with this configuration:

<VirtualHost *:80>
    ServerName orari.oratoridisettimo.it
    DocumentRoot "/var/www/html/orari.mydomain.it/html/public"
    <Directory /var/www/html/orari.mydomain.it/html/public>
        AllowOverride All
        Require all granted
        FallbackResource /index.php
    </Directory>
    <Directory /var/www/html/orari.mydomain.it/html>
        Options FollowSymlinks
    </Directory>
    <Directory /var/www/html/orari.mydomain.it/html/public/bundles>
        FallbackResource disabled
    </Directory>
    ErrorLog /var/log/kimai_error.log
    CustomLog /var/log/kimai_access.log combined
</VirtualHost>

I removed kimai2 folder between "html" and "public" and I reported the correct path in the DocumentRoot. Still not working

Apache 2.4
PHP 7.4

**Reply by TechBreakHelp** [reply]
Solved. Apache has created a config file called /etc/apache2/sites-available/timetracker.mydomain.com-le-ssl.conf which modifies the configuration. It is used for port 443 (SSL) but it hasn't got an SSL certificate path. I modified that file and now it works properly

**Reply by kevinpapst** [reply]
Glad you found it. Thanks for your feedback, now I hope Kimai is useful for you!

**Reply by TechBreakHelp** [reply]
Yes, sure! Hope this will help someone else too!
